front/src/applications/stdcm/components/SimulationReportSheetV2.tsx

```diff
@@ -10,21 +10,19 @@ import { formatDateToString, formatDayV2 } from 'utils/date';
 
 import styles from './SimulationReportStyleSheet';
 import type { SimulationReportSheetProps } from '../types';
-import { extractSpeedLimit, getStopDurationTime, getOperationalPointsWithTimes } from '../utils';
+import { extractSpeedLimit, getStopDurationTime } from '../utils';
 
 const SimulationReportSheetV2 = ({
   stdcmData,
-  pathProperties,
-  rollingStockData,
-  speedLimitByTag,
   simulationReportSheetNumber,
   mapCanvas,
-  creationDate,
+  operationalPointsList,
 }: SimulationReportSheetProps) => {
   const { t } = useTranslation('stdcm-simulation-report-sheet');
   let renderedIndex = 0;
 
-  const date = creationDate && t('formattedDate', formatDateToString(creationDate));
+  const date =
+    stdcmData.creationDate && t('formattedDate', formatDateToString(stdcmData.creationDate));
```

Comment:
With ts, you know that creationDate is defined, so you can remove the test. Since it only remains the translation, I think you can directly add it in the html :)
```suggestion
  const date =
    t('formattedDate', formatDateToString(stdcmData.creationDate));
```

Reply:
fix